docs/dev/debugging.md

```diff
@@ -1,44 +1,67 @@
 # Debugging vs Code plugin and the Language Server
 
-**NOTE:** the information here is mostly obsolete
+## Prerequsities
 
-Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).
+- Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) VSCode extensions.
+- Open the root folder in VSCode. Here you can access the preconfigured debug setups.
 
-Checkout rust rust-analyzer and open it in vscode.
+  <img height=150px src="https://user-images.githubusercontent.com/36276403/74611090-92ec5380-5101-11ea-8a41-598f51f3f3e3.png" alt="Debug options view">
 
-```
-$ git clone https://github.com/rust-analyzer/rust-analyzer.git --depth 1
-$ cd rust-analyzer
-$ code .
-```
+- Install all TypeScript dependencies
+  ```bash
+  cd editors/code
+  npm i
+  ```
 
-- To attach to the `lsp server` in linux you'll have to run:
+## Common knowledge
+
+* All debug configurations open new `[Extension Development Host]` VSCode instance
```

Comment:
```suggestion
* All debug configurations open a new `[Extension Development Host]` VSCode instance
 where **only** the `rust-analyzer` extension being debugged is enabled. 
```

Reply:
Thank you, articles are my weak point :D

Reply:
I had no idea that Russian and most related languages had no articles :smile:.

Reply:
TYL )

Reply:
I suppose you can select multiple lines to edit, because we need to amend the second line it commit suggsetion too

Reply:
Sorry, I'll let you do that locally. I tried it a couple of PRs ago and it didn't turn out too well.